prospero-cli/src/main/resources/UsageMessages.properties

```diff
@@ -74,6 +76,8 @@ prospero.completion.usage.description.8 = \u0020  ${PARENT-COMMAND-FULL-NAME:-$P
 prospero.completion.usage.description.9 = \u0020  echo \". ~/${prospero.dist.name}_completion\" >> ~/.bashrc
 prospero.completion.usage.description.10 =
 
+prospero.apply-update.usage.header = Apply previously prepared update to the server. The server has to be stopped during this operation.
```

Comment:
Thinking out loud - are we able to detect server is running? Are we doing it?

Reply:
Yes, it's the same mechanic Galleon is using: https://github.com/wildfly-extras/prospero/blob/1faa119ab5c220299fe4d5d4d0019580ffe35ba2/prospero-common/src/main/java/org/wildfly/prospero/actions/ApplyUpdateAction.java#L160